llvm/lib/ObjectYAML/DXContainerYAML.cpp

```diff
@@ -24,14 +24,14 @@ static_assert((uint64_t)dxbc::FeatureFlags::NextUnusedBit <= 1ull << 63,
               "Shader flag bits exceed enum size.");
 
 DXContainerYAML::ShaderFlags::ShaderFlags(uint64_t FlagData) {
-#define SHADER_FLAG(Num, Val, Str)                                             \
+#define SHADER_FEATURE_INFO(Num, Val, Str)                                     \
```

Comment:
I think this renaming makes this more confusing not less. We probably have a problem that we have too many things called "shader flags", but only partially changing the name makes this more confusing.

A name like "shader feature flags" might be a better name and could be differentiated from the "shader module flags" that are present in the DXIL metadata, but if we're going to do a renaming it should be comprehensive and not leave us in a confusing state.

Reply:
Updated to SHADER_FEATURE_FLAG